src/interface/index.ts

```diff
@@ -40,13 +40,24 @@ export enum ModalName {
  * Use to identify low-level components given a TraceContext
  */
 export enum ElementName {
+  ABOUT_PAGE_DOCS_BUTTON = 'about-page-docs-button',
+  ABOUT_PAGE_FOOTER_BLOG_BUTTON = 'about-page-blog-button',
+  ABOUT_PAGE_FOOTER_CAREERS_BUTTON = 'about-page-careers-button',
+  ABOUT_PAGE_FOOTER_SUPPORT_BUTTON = 'about-page-footer-support-button',
+  ABOUT_PAGE_FOOTER_TWITTER_BUTTON = 'about-page-twitter-button',
+  ABOUT_PAGE_LEARN_MORE_BUTTON = 'about-page-learn-more-button',
+  ABOUT_PAGE_NFTS_CARD = 'about-page-nfts-card',
+  ABOUT_PAGE_SWAP_CARD = 'about-page-swap-card',
```

Comment:
The page prefix should be captured as part of the Trace context 🤔 Are we sure we want these prefixes here? (Same for the landing page)

Reply:
oh i can remove sorry

Reply:
@dannydaniil if i don't do that then the names will be very confusing? i could make it `ABOUT_PAGE_DOCS_BUTTON = 'docs-button',` if you want?

Reply:
@vm Since the trace context will capture the page is there an issue with going general and reusing?

Some of these seem more specific like `landing-page-swap-element` where `swap-element` is ambiguous, but I think the rest may end up being reusable (e.g. footer careers button I can imagine being elsewhere in the future).

I'm good with either full general or otherwise keeping the full prefix but mixing probably isn't helpful. Up to you IMO as long as we're consistent at minimum between products (e.g. interface in this case)!

Reply:
okay lets make them general then